how-to/create-working-group.md

```diff
@@ -0,0 +1,9 @@
+This document describes some part of how to create a working group in the InnerSource Commons.
+
+* Create a _Slack_ channel with the name of your working group.
+* Send out invites using the [InnerSource Commons calendar].
+* Schedule video calls with https://meet.jit.si/.
+* Make a folder with the name of your working group in the [InnerSource Commons GDrive].
```

Comment:
Should we mention that they have to request access to this?

Reply:
is meet.jit.si the tool used from now by all ISC events?

Reply:
> Should we mention that they have to request access to this?

Yes.  How does one request access?

Reply:
> Should we mention that they have to request access to this?

Looks like no request is needed.

![image](https://user-images.githubusercontent.com/9609562/220944128-c156f7e3-83d1-45cc-b170-f006f2c3f761.png)